### ISSUE TITLE
Orca does not appear to exist in the installer

### ISSUE THREAD

**Issue by devinprater** [manjaro/release-plan#208]
In the Manjaro installer, [Orca](https://github.com/GNOME/orca) cannot be started. I downloaded the Mate edition, since Mate has greater accessibility. I etched the installer onto USB, plugged it in, and, with sighted assistance, got the installer open.

I tried using Alt + Super + S, which usually starts Orca, then Alt + F2, typed orca, then pressed Enter. Orca wasn't found. I then launched the terminal, ran orca, with the same result. As a blind person, I'd love to use Manjaro, as it seems to be an easy way to install Arch. I'd greatly appreciate the inclusion of Orca into the installer, configured to launch on the press of Alt + Super + s.

If this isn't the correct repository to post this issue, please let me know.

**Comment by Ste74** [comment]
Unfortunately at the moment, Calamares ( our GUI installer ) not support Orca etc.. Here is the actual state of art https://github.com/calamares/calamares/issues/1294
